coder_upgrade.module

```diff
@@ -183,5 +184,5 @@ function coder_upgrade_build_config_function($node, $reader) {
     'type' => T_WHITESPACE,
     'value' => 1,
   );
-  $container->insertBefore($node, $whitespace, 'whitespace');
```

Comment:
Need to delete the array defnition just above.

Reply:
Thx, will do.